data/fields/parking/parking_side_orientation.json

```
@@ -0,0 +1,14 @@
+{
+    "key": "parking:$side:orientation",
+    "type": "combo",
+    "label": "Type",
+    "strings": {
+        "options": {
+            "parallel": "Parallel to the Street",
+            "diagonal": "Diagonal in Relation to the Street",
+            "perpendicular": "Meets the Street at a Straight Angle"
+        }
+    },
+    "autoSuggestions": false,
```

I noticed in iD, that when I add `population`, that also the `population:source` and `check_date:population` is added to the fields automatically. I think that feature would be a good fit for this key as well. So I guess we need to add …

```suggestion
    "prerequisiteTag": {
        "key": "parking:$side"
    },
    "autoSuggestions": false,
```